scripts/generate_screenshots.py

```
@@ -0,0 +1,124 @@
+        header.append(f"  {r['file_url']}", style="dim")
+        console.print(header)
+        for m in r["matches"]:
+            line_text = Text("  ")
+            line_text.append(f"L{m['line']:>5}", style="green")
+            line_text.append(f"  {m['preview']}")
+            console.print(line_text)
+        console.print()
+    console.print(f"[dim]{STATS_LINE}[/dim]")
+
+
+def make_svg(render_fn, title: str) -> str:
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)